### DISCUSSION TITLE
Global operation Lock for HmIP-eTRV-E

### DISCUSSION THREAD

**Discussion by ttnotet** [SukramJ/aiohomematic#1247] [Q&A] [answered]
It would be good to diplay and switch the Global Operation Lock of the HmIP-eTRV-E.

**Comment by SukramJ** [comment, marked as answer]
Add  `GLOBAL_BUTTON_LOCK@HmIP-eTRV-E:0:MASTER` to [unignore](https://github.com/danielperna84/custom_homematic#unignore-device-parameters).
